packages/runtime/container-runtime/package.json

```diff
@@ -120,6 +120,9 @@
 			},
 			"ClassDeclaration_ContainerRuntime": {
 				"forwardCompat": false
+			},
+			"InterfaceDeclaration_IConnectableRuntime": {
+				"backCompat": false
```

Comment:
This is a breaking change.  I think you just have to mark it as deprecated.  Probably mark the whole `IConnectableRuntime` interface as deprecated and remove it from exports in `next`, at least it's not used in Bohemia codebase.

Reply:
I tried removing it but it adds a "missing export" warning because ContainerRuntime implements ISummarizerRuntime and ContainerRuntime is exported.